cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -48,6 +48,40 @@ data:
         tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
 `
 
+	// KubeProxyConfigMap is the proxy ConfigMap manifest
+	KubeProxyConfigMap19 = `
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-proxy
+  namespace: kube-system
+  labels:
+    app: kube-proxy
+data:
+  kubeconfig.conf: |-
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - cluster:
+        certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        server: {{ .MasterEndpoint }}
+      name: default
+    contexts:
+    - context:
+        cluster: default
+        namespace: default
+        user: default
+      name: default
+    current-context: default
+    users:
+    - name: default
+      user:
+        tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+  config.conf: |-
+    apiVersion: kubeproxyconfig/v1alpha1
+    kind: KubeProxyConfiguration
+{{ .ProxyConfig}}
```

Comment:
doesn't `{{ .ProxyConfig | indent 4 }}` cut it?

Reply:
I couldn't get it to work, it doesn't seem to be part of the standard go library
see - https://play.golang.org/p/aVrYQ9Yt9e
The examples in kubernetes/charts seem to be all yaml files.

Reply:
Ok, sorry, I didn't see that you pointed this out in the main comments already...